test/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+set(SRCROOT "${PROJECT_SOURCE_DIR}/test/src")
+
+include_directories("${PROJECT_SOURCE_DIR}/include")
+include_directories("${PROJECT_SOURCE_DIR}/extlibs/headers")
+include_directories("${SRCROOT}/TestUtilities")
+
```

Comment:
Why not putting each test target in its own CMakeLists.txt file, in the corresponding sub-folder?

Reply:
Might be part of a future enhancement, for now I think we'll just stick to this.